independent-publisher-connectors/ORB Intelligence/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4731 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ORB Database",
+    "version": "v3",

```

Comment:
@aaryan2134 This can be only "1.0.0" or "1.0" for new connectors.

Reply:
I tried doing this but I am getting errors due to this. Is there some way I can convert 2.0 file to 1.0?

Reply:
@sriyen-msft thank you for your feedback sir we tried but we are getting errors could you suggest any resources that we can use to fix this issue

Reply:
@sriyen-msft was talking about connector version, NOT swagger version. @AnkitaSinghIE your connector version should look like this:
```
"info": {
    "title": "ORB Database",
    "version": "1.0",
```

Reply:
@makaveli1313 Oh. Thank you so much for the clarification!

Reply:
Done

Reply:
@makaveli1313 , thank you, sir. We made the changes could you please review it